Drv/StreamCrossover/StreamCrossover.cpp

```diff
@@ -0,0 +1,50 @@
+// ======================================================================
+// \title  StreamCrossover.cpp
+// \author ethanchee
+// \brief  cpp file for StreamCrossover component implementation class
+// ======================================================================
+
+
+#include <Drv/StreamCrossover/StreamCrossover.hpp>
+#include <FpConfig.hpp>
+
+namespace Drv {
+
+  // ----------------------------------------------------------------------
+  // Construction, initialization, and destruction
+  // ----------------------------------------------------------------------
+
+  StreamCrossover ::
+    StreamCrossover(
+        const char *const compName
+    ) : StreamCrossoverComponentBase(compName)
+  {
+
+  }
+
+  StreamCrossover ::
+    ~StreamCrossover()
+  {
+
+  }
+
+  // ----------------------------------------------------------------------
+  // Handler implementations for user-defined typed input ports
+  // ----------------------------------------------------------------------
+
+  void StreamCrossover ::
+    streamIn_handler(
+        const NATIVE_INT_TYPE portNum,
+        Fw::Buffer &recvBuffer,
+        const Drv::RecvStatus &recvStatus
```

Comment:
Receive status is not being checked.  We should probably check that and not send through if the status is bad.